posthog/async_migrations/definition.py

```diff
@@ -40,9 +40,6 @@ def __init__(
         # Defaults to a no-op ("") - None causes a failure to rollback
         self.rollback_fn = rollback_fn
 
-        # If the operation specifies raw SQL, add this as a property for easier debugging
-        self.sql = sql
```

Comment:
I'm still strongly opposed to this. We have 2 migrations generating SQL dynamically and we don't have that SQL easily available anywhere. Even with the `current_operation_index` it's very hard to determine what the SQL for the operation is without this or something similar. 

We can rename to `_sql` or even `__sql` if it makes you more comfortable